Be consistent with DATABASE_URL value

vjousse opened the pull request:
Above in the documentation, we are using `postgresql://<username>:<password>@127.0.0.1/<appname>_production` for the `DATABASE_URL` value. I think we should be consistent here too as it's the format that will work for most people.

vjousse committed:
Be consistent with DATABASE_URL

Above in the documentation, we are using `postgresql://<username>:<password>@127.0.0.1/<appname>_production` for the `DATABASE_URL` value. I think we should be consistent here too as it's the format that will work for most people.

jwoertink commented on src/actions/guides/deploying/ubuntu.cr:
```diff
@@ -167,7 +167,7 @@ class Guides::Deploying::Ubuntu < GuideAction
     Environment="LUCKY_ENV=production"
     Environment="SECRET_KEY_BASE=<random unique key>"
-    Environment="DATABASE_URL=postgres:///<yourapp>_production"
+    Environment="DATABASE_URL=postgresql://<username>:<password>@127.0.0.1/<appname>_production"
```
Interesting. I've always done `postgres://`, but I just read 

> The URI scheme designator can be either postgresql:// or postgres://.

https://www.postgresql.org/docs/12/libpq-connect.html

I'm thinking maybe we should just keep it consistent with what Avram uses [here](https://github.com/luckyframework/avram/blob/master/src/avram/credentials.cr#L132) 🤔  Can you grep through the rest of the docs and see if we mix this up? If we use both, then lets roll with the shorter one, but I like the rest of the change here 👍

vjousse replied:
It's `postgres` that is used everywhere in the docs. I've updated the PR accordingly :+1:

vjousse committed:
Merge branch 'master' of github.com:luckyframework/website into patch-3

vjousse committed:
Use `postgres` instead of `postgresql` DB URL

Let's be consistent with what avram is using here: https://www.postgresql.org/docs/12/libpq-connect.html

paulcsmith reviewed (approved):
This looks great. Thanks @vjousse
